CHANGELOG.md

```diff
@@ -11,6 +16,7 @@
 ### Fixes
 
 - **Made `Button` children optional** to allow using buttons with only icons.
+- **Fix `Slider` bug where onChange was not always called after pointerUp.**
```

Comment:
huh?

Reply:
Bad merge? 
![image](https://user-images.githubusercontent.com/36186912/127450748-d2242f3e-cdb5-4127-97dd-dfee4b4680c9.png)

Reply:
what

Reply:
I fixed, didnt see at first, cause was hidden

Reply:
But it auto merged, no conflicts..